compliance/collection/nodeinventorizer/nodeinventory.go

```diff
@@ -78,3 +92,13 @@ func convertRHELComponents(rc *database.RHELv2Components) []*scannerV1.RHELCompo
 	}
 	return v1rhelc
 }
+
+func equalRHELv2Packages(a *database.RHELv2Package, b *database.RHELv2Package) bool {
+	if a == nil || b == nil {
+		return false
+	}
+	if a.Name == b.Name && a.Version == b.Version && a.Arch == b.Arch && a.Module == b.Module {
+		return true
+	}
+	return false
```

Comment:
```suggestion
	return a.Name == b.Name && a.Version == b.Version && a.Arch == b.Arch && a.Module == b.Module
```

Reply:
Nit: If we keep this (not necessary if we use a map): is it OK to break in different lines? Easier to tell what we are comparing to someone reading it.